src/main/kotlin/lotto/domain/LottoSeller.kt

```diff
@@ -1,16 +1,26 @@
 package lotto.domain
 
+import lotto.io.InputView
 import lotto.util.LottoGenerator
+import lotto.util.ManualLottoGenerator
+import lotto.util.RandomLottoGenerator
 
-class LottoSeller(private val lottoGenerator: LottoGenerator) {
+object LottoSeller {
+    private val manualLottoGenerator = ManualLottoGenerator { InputView.getManualLotto().map { LottoNumber(it) } }
```

Comment:
LottoGame에서 View 계층과 통신하고 있으니 LottoGame에서 LottoSeller의 생성자에 고차 함수를 주입하는 방식은 어떨까요?
```suggestion
class LottoSeller(private val manualLottoCommand: () -> List<LottoNumber>) {
    private val manualLottoGenerator = ManualLottoGenerator(manualLottoCommand)
```